src/wipplpy/mst/connection.py

```diff
@@ -0,0 +1,73 @@
+"""
+Define the MSTConnection() class specifically for making MST-MDSplus connection
+requests. 
+"""
+
+# Standard
+from datetime import date
+import math
+
+# Local
+from WiPPLPy.modules.config_reader import ConfigReader
+from WiPPLPy.modules.connection import MDSPlusConnection
+
+
+class MSTConnection(MDSPlusConnection):
+
+    def __init__(self):
+        """
+        Initialize MST-relevant objects that are necessary for connecting to
+        MDSplus.
+        """
+        self.remote_server = ConfigReader.MST_remote_server 
+        self.local_server = ConfigReader.MST_local_server 
+        self.servers_dictionary = {
+                True: self.remote_server,
+                False: self.local_server}
+        self.tree_name = ConfigReader.MST_tree 
+
+    @staticmethod
+    def data_location(shot_number):
+        """
+        Determine the location (server) of a shot's data.
+
+        Parameters
+        ----------
+        shot_number : `int`
+            The shot-number from which to extract MDSplus data.
```

Comment:
Do we want to do `shot-number` or `shot number` or `shotnumber`? We should choose one and stick to it.

Reply:
We can keep it as 'shot number.'